core/common/assets/js/views/modal/header.js

```diff
@@ -37,6 +37,8 @@ export default class extends Marionette.LayoutView {
 		this._parent._parent._parent.hideModal();
 
 		if ( this.isFloatingButtonLibraryClose() ) {
+			// Had to use jQuery as it is the only thing that worked.
```

Comment:
Please remove this comment

Reply:
@nicoladj77 did it not work to autosave?

Reply:
@louiswol94 no didn't work.